pkg/agent/openflow/client.go

```diff
@@ -1215,7 +1251,8 @@ func (c *client) InstallMulticastGroup(groupID binding.GroupIDType, localReceive
 		table = MulticastIngressRuleTable
 	}
 
-	if err := c.featureMulticast.multicastReceiversGroup(groupID, table.GetID(), localReceivers...); err != nil {
+	targetPorts := append([]uint32{config.HostGatewayOFPort}, localReceivers...)
```

Comment:
gateway port is already included in the localReceivers

Reply:
updated